src/main/java/net/countercraft/movecraft/combat/movecraftcombat/config/Config.java

```diff
@@ -62,4 +62,6 @@
 
     // Add fires to craft hitbox
     public static boolean AddFiresToHitbox = true;
+
+    public static boolean ReImplementTranslocation = false;
```

Comment:
This should be renamed to ReImplementTNTTranslocation, since it is not general translocation.